### PR TITLE
fix: chat server standalone deployment and update dependencies - EXO-62759

### PR DESCRIPTION
**Pull request by bourasom**
Before this PR, the chat application server standalone stucks on startup due to missing ch.qos.logback  and org.codehaus.janino dependencies' versions resolution. Besides, the used tomcat version was 8.5.56 and the mongo driver's was 3.5.0.
This PR introduce missing dependencies and upgrade tomcat and mongo driver's

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -43,15 +43,18 @@
...
     <aopalliance.version>1.0</aopalliance.version>
     <com.google.json-simple.version>1.1.1</com.google.json-simple.version>
 
     <!-- The apache tomcat version to bundle -->
-    <org.apache.tomcat.version>8.5.65</org.apache.tomcat.version>
+    <org.apache.tomcat.version>9.0.60</org.apache.tomcat.version>
     <quartz.version>2.2.2</quartz.version>
-    <org.slf4j.version>1.7.7</org.slf4j.version>
+    <org.slf4j.version>1.7.36</org.slf4j.version>
+    <ch.qas.logback.version>1.2.6</ch.qas.logback.version>
```
Comment (ahamdi) [inline pom.xml]: to fix Snyk check
```suggestion
    <ch.qas.logback.version>1.2.7</ch.qas.logback.version>
```

**Review by ahamdi — COMMENTED** [review]
@bourasom Shouldn't this go through maintenance for testing ?

**Comment by bourasom** [comment]
> @bourasom Shouldn't this go through maintenance for testing ?

Yes for sure. Shall I add a PR on maintenance fb?

**Review by rdenarie — DISMISSED** [review]

**Comment by rdenarie** [comment]
> > @bourasom Shouldn't this go through maintenance for testing ?
> 
> Yes for sure. Shall I add a PR on maintenance fb?

not on maintenance branche but on stable/3.4.x, to have it in 6.4.0, no ?

**Review by ahamdi — APPROVED** [review]

**Pull request merged**